src/js/plugins/plugin.vendor-fullscreen.js

```diff
@@ -0,0 +1,231 @@
+/* global BookReader */
+
+/**
+ * Toggles browser's native fullscreen mode if available
+ */
+
+const EVENT_NAMESPACE = '.bookreader_vendor-fullscreen';
+// function isMobile() {
+//   return (typeof window.orientation !== 'undefined') || (navigator.userAgent.indexOf('IEMobile') !== -1);
+// };
+
+// if (this.isMobile()) { return; }
```

Comment:
I don't think we can comment this out; for backward compatibility, let's put the whole file in an `if` block:

```js
if (!isMobile()) {
  BookReader.prototype....
}
```

Reply:
Basically its current WIP pull request.